include/revng/PTML/ModelEditPath.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/DynamicFunction.h"
+#include "revng/Model/Function.h"
+#include "revng/Model/Segment.h"
+#include "revng/PTML/Tag.h"
+
+namespace modelEditPath {
```

Comment:
* [ ] Rename to `ModelHelpers.h
* [ ] Rename `create` to `getCustomNamePath`

Reply:
Done